lib/Controller/PageController.php

```diff
@@ -72,4 +72,25 @@ public function context(int $contextId): TemplateResponse {
 
 		return $this->index();
 	}
+
+	#[PublicPage]
+	#[NoCSRFRequired]
+	#[OpenAPI(scope: OpenAPI::SCOPE_IGNORE)]
+	#[FrontpageRoute(verb: 'GET', url: '/s/{token}')]
+	public function linkShare(string $token): TemplateResponse {
+		Util::addScript(Application::APP_ID, 'tables-main');
+		$this->loadStyles();
+
+		$shareToken = new ShareToken($token);
+		$this->initialState->provideInitialState('shareToken', (string)$shareToken);
```

Comment:
To properly load text editor so that rich text cells display properly, we need to add:
```
		if (class_exists(LoadEditor::class)) {
			$this->eventDispatcher->dispatchTyped(new LoadEditor());
		}
```

I already did at https://github.com/nextcloud/tables/pull/2236/changes#diff-70e5309fcd3311d771b3db9c93490ea270fd5894769093765ec37edb68e5dd9b